cvat/apps/dataset_repo/views.py

```diff
@@ -85,6 +86,35 @@ def get_repository(request, tid):
 
         return HttpResponseBadRequest(str(ex))
 
+@login_required
+@permission_required(perm=['engine.task.access'],
```

Comment:
@PMazarovich , the file doesn't follow the new permission system. I will merge it as is and create an issue to update the app in accordance with IAM in the PR: https://github.com/openvinotoolkit/cvat/pull/3788.

Reply:
@PMazarovich , please let us know if you can help us to update the Django app.